conversions/octal_to_hexadecimal.c

```diff
@@ -0,0 +1,72 @@
+/**
+ * @brief Octal to hexadecimal conversion by scanning user input
+ * @details
+ * The octalToHexadecimal function take the octal number as long
+ * return a string hexadecimal value after conversion
+ * @author [Rachit Bhalla](https://github.com/rachitbhalla)
+ */
+#include <assert.h>
+#include <math.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
```

Comment:
Provide a brief description of the headers (what to they add).
```suggestion
#include <assert.h>  /// for assert
#include <math.h>    /// 
#include <stdio.h>  ///
#include <stdlib.h>  ///
#include <string.h>  ///
```

Reply:
Updated! Pls verify.